packages/amplify_auth_plugin_interface/lib/src/Attribute/AuthUserAttributeOptions.dart

```diff
@@ -12,10 +12,8 @@
  * express or implied. See the License for the specific language governing
  * permissions and limitations under the License.
  */
-class AuthUserAttributeOptions {
-  const AuthUserAttributeOptions();
-  Map<String, dynamic> serializeAsMap() {
-    throw UnimplementedError(
-        'serializeAsMap() has not been implemented on AuthUserAttributeOptions.');
-  }
-}
+
+import 'FetchUserAttributesOptions.dart';
+
+@Deprecated("Please use 'FetchUserAttributesOptions' instead")
```

Comment:
nit: should use backtick to wrap class name?

Reply:
I mentioned this on another PR, but I am going to update these to follow what seems to be the pattern in the flutter/flutter repo, which is to not wrap functions/types in anything.

Reply:
```suggestion
@Deprecated('Please use FetchUserAttributesOptions instead')
```